sale_line_refund_to_invoice_qty/models/account.py

```diff
@@ -12,15 +12,17 @@ def _reverse_move_vals(self, default_values, cancel=True):
         move_vals = super(AccountMove, self)._reverse_move_vals(
             default_values, cancel=cancel
         )
-        if self.env.context.get("sale_qty_to_reinvoice", False):
-            for vals in move_vals["line_ids"]:
-                vals[2].update({"sale_qty_to_reinvoice": True})
+        sale_qty_to_reinvoice = self.env.context.get("sale_qty_to_reinvoice", False)
+        for vals in move_vals["line_ids"]:
+            vals[2].update({"sale_qty_to_reinvoice": sale_qty_to_reinvoice})
         return move_vals
 
 
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     sale_qty_to_reinvoice = fields.Boolean(
+        string="Sale qty to reinvoice",
```

Comment:
this is not necessary.
```suggestion
```
https://github.com/OCA/OCB/blob/16.0/odoo/fields.py#L507

Reply:
Done!